src/containers/exchange/simplebuy.js

```diff
@@ -0,0 +1,36 @@
+export function matchOrders(n, orders, direction) {
```

Comment:
This file and the corresponding test case should be deleted (the functionality is in augmint.js)